hammocking/hammocking.py

```diff
@@ -171,7 +173,7 @@ def done(self) -> bool:
 
 
 class NmWrapper:
-    regex = r"\s*U\s+([^_]\S*)"
```

Comment:
LLVM emits code coverage symbols like llvm_gcov_init, llvm_gcda_emit_arcs, that are unmockable.